elevate-cpanel

```diff
@@ -1855,6 +1855,24 @@ EOS
     use constant YUM_COMPLETE_TRANSACTION_BIN => '/usr/sbin/yum-complete-transaction';
     use constant FIX_RPM_SCRIPT               => '/usr/local/cpanel/scripts/find_and_fix_rpm_issues';
 
+    use constant EXPECTED_EXTRA_PACKAGES => (
+        qr/^cpanel-/,
+        qr/^easy-/,
+        qr/^kernel/,
+        qr/^mysql/i,
+        qr/^plesk-/,
+        'awscli',
+        'basesystem',
+        'filesystem',
+        'grub',
+        'grubby',
+        'python35',
+        'python38-opt',
+        'virt-what',
+        'vzdummy-systemd-el7',
+      ),
+      Elevate::Constants::R1SOFT_AGENT_PACKAGES;
+
```

Comment:
Awesome I was contemplating the best way to go about the allowed / disallowed packages.

Not sure if you want to revisit https://webpros.atlassian.net/issues/RE-597?jql=text%20~%20%22awscli%22%20AND%20project%20%3D%20RE in another task to handle those cases since this would allow it so easily.